__tests__/app.test.js

```diff
@@ -81,6 +82,48 @@ describe("GET:/api/articles/:article_id", () => {
   });
 });
 
+describe("GET:/api/articles/:article_id/comments", ()=>{
+  test("Get 200: returns an array of comments for the given article_id",()=>{
+    return request(app).get("/api/articles/3/comments").expect(200).then((response) => {
+      const comments = response.body
+      
+      expect(comments).toHaveLength(2)
+      expect(comments).toBeSortedBy("created_at", {descending:true})
+      comments.forEach((comment) => {
+        expect(comment).toHaveProperty("comment_id");
+        expect(comment).toHaveProperty("votes");
+        expect(comment).toHaveProperty("created_at");
+        expect(comment).toHaveProperty("author");
+        expect(comment).toHaveProperty("body");
+        expect(comment).toHaveProperty("article_id")
```

Comment:
You can make these test stronger by also looking at value (not just the key) and ensuring it has the right data type.

for example...
`        expect(comment).toHaveProperty("comment_id", expect.any("Number"));`
What you put after the key is what you're expecting it to be.
We can use this also to make sure all the article ids are 3, since the request is requesting comments on the article with an ID of 3